packages/lib/src/toolbar/OverflowMenu.tsx

```diff
@@ -1,17 +1,17 @@
 import { useClickOutside, useToggle } from '@react-hookz/web';
-import type { ReactElement } from 'react';
-import { Children, cloneElement, useRef } from 'react';
+import type { PropsWithChildren } from 'react';
+import { cloneElement, isValidElement, useRef } from 'react';
 import { FiMenu } from 'react-icons/fi';
+import flattenChildren from 'react-keyed-flatten-children';
 
 import styles from './OverflowMenu.module.css';
 import Separator from './Separator';
 
-interface Props {
-  children: ReactElement[];
-}
+interface Props {}
 
-function OverflowMenu(props: Props) {
+function OverflowMenu(props: PropsWithChildren<Props>) {
```

Comment:
When debugging the error in Braggy, I thought the issue came from here at first. I refactored the type of the `children` prop to be more lenient. In summary, I allow any children to be passed to `OverflowMenu`, which simplifies the JSX code in `Toolbar`, but then I make sure to take into account only valid `ReactElement` children below.